tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -4247,22 +4224,24 @@ public function cleanupDatabaseGroups():void {
 		$previousServer = $this->currentServer;
 		$this->usingServer('LOCAL');
 		foreach ($this->createdGroups as $group => $groupData) {
-			if ($this->isTestingWithLdap()) {
-				$this->cleanupGroup((string)$group);
-			} else {
-				$this->graphContext->adminDeletesGroupWithGroupId(
-					$groupData['id']
-				);
+			if ($groupData["possibleToDelete"]) {
+				if ($this->isTestingWithLdap()) {
+					$this->cleanupGroup((string)$group);
+				} else {
+					$response = $this->graphContext->deleteGroupWithId($groupData['id']);
+					$this->theHTTPStatusCodeShouldBe(204, "", $response);
```

Comment:
I wonder if we want to cause `cleanupDatabaseGroups` to fail if the group delete gets a 404 (if the group already does not exist).
But I suppose it is good to know if this happens - it means that some test code has created a group that didn't get put into the `createdGroups` array.